osmosis-1/osmosis-1.chainlist.json

```diff
@@ -5565,6 +5565,64 @@
         "ibc-transfer",
         "ibc-go"
       ]
+    },
+    {
+      "chain_name": "fxcore",
+      "status": "live",
+      "network_type": "mainnet",
+      "pretty_name": "f(x)Core",
+      "chain_id": "fxcore",
+      "bech32_prefix": "fx",
+      "bech32_config": {
+        "bech32PrefixAccAddr": "fx",
+        "bech32PrefixAccPub": "fxpub",
+        "bech32PrefixValAddr": "fxvaloper",
+        "bech32PrefixValPub": "fxvaloperpub",
+        "bech32PrefixConsAddr": "fxvalcons",
+        "bech32PrefixConsPub": "fxvalconspub"
+      },
+      "slip44": 60,
+      "fees": {
+        "fee_tokens": [
+          {
+            "denom": "FX",
+            "fixed_min_gas_price": 4200000000000,
+            "low_gas_price": 400000000000,
+            "average_gas_price": 4200000000000,
```

Comment:
Please correct these prices at the Chain Registry because they don't make sense. The average is higher than the high and the chain minimum is higher than the low price.

Reply:
PRed to chain registry here https://github.com/cosmos/chain-registry/pull/2794